source/settings/settings-reference.txt

```diff
@@ -107,6 +107,15 @@ You can configure the following settings on the |compass| interface:
        enabled. You cannot toggle this option on Linux. You can toggle this 
        setting only on Windows or macOS.
 
+   * - Show Quit Confirmation Dialog
+     - General
+     - Toggle whether a confirmation dialog is shown when quitting Compass.
+       Checking the ``Do not ask again`` checkbox on the quit dialog 
+       disables this setting.
```

Comment:
(structure)
To disable this setting, check "Do not ask again".

Reply:
This removes the reference to the quit dialog box which is a separate element than where the setting is stored. This is a supplemental way to turn off the setting option. 

So it needs to be:

"You can also check the ``Do not ask again`` checkbox on the quit dialog to disable the setting."